review.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// Conduct a brief Code Review of the given code snippets in the context of the previous Task
+
+/*
+ * The function storyStats returns four things:
+ * the shortest word
+ * the longest word
+ * the average word length
+ * the list of words that are of length equal to the average word rounded up and down, or empty list if such words do not exist.
+ */
+func storyStats1(str string) (map[string]string, error) {
+	var err error
+	m := make(map[string]string)
+	m["shortest"] = ""
+	m["longest"] = ""
+	m["average"] = ""
```

Comment:
weak approach, use struct instead

Reply:
missing "list of words that are of length equal to the average word rounded up and down, or empty list if such words do not exist."